src/neural/writer.h

```diff
@@ -44,9 +46,13 @@ struct V5TrainingData {
   uint8_t castling_us_oo;
   uint8_t castling_them_ooo;
   uint8_t castling_them_oo;
-  uint8_t side_to_move;
+  // For input type 3 contains enpassant column as a mask.
+  uint8_t side_to_move_or_enpassant;
   uint8_t rule50_count;
-  uint8_t deprecated_move_count;  // left in to keep 8 int8 fields.
+  // For input type 3 contains a bit field indicating the transform that was
+  // used and the original side to move info.
+  // Side to move is in the top bit, transform in the lower bits.
+  uint8_t deprecated_move_count_or_invariance_info;
```

Comment:
Why not just rename it to `invariance_info`? Maybe also add a comment that it used to be the `deprecated_move_count` that was kept for padding.